libraries/amqp/include/eosio/amqp/amqp_handler.hpp

```diff
@@ -230,7 +230,7 @@ class amqp_handler {
          first_connect_.set();
       } );
       consumer.onReceived( [&](const AMQP::Message& message, const delivery_tag_t& delivery_tag, bool redelivered) {
-         on_consume_( delivery_tag, message.body(), message.bodySize() );
+         on_consume_( message, delivery_tag, redelivered );
```

Comment:
`on_consume_t` is almost the same type as `AMQP::MessageCallback` now.  Is there a reason for forwarding the arguments through an extra lambda?

Reply:
`onReceived` takes a std::function. So no need for the extra lambda.